eventually/tests/unittests/authentication/testforgetpassword.py

```diff
@@ -0,0 +1,114 @@
+"""
+Forget Password test module
+===========================
+"""
+
+import json
+
+from django.test import TestCase, Client
+
+from authentication.views import CustomUser
+from utils.jwttoken import create_token
+
+VALID_EMAIL_TOKEN = create_token({"user_id":100, 'email': 'john@email.com'}, 60 * 60)
```

Comment:
maybe it is not a good idea to create the token with appropriate custom util method. If that method is wrong the test result would be incorrect.

Reply:
Sorry, forgot about that. Fixed now.